oracle/controllers/backupcontroller/backup_controller_test.go

```diff
@@ -108,11 +108,13 @@ var _ = Describe("Backup controller", func() {
 	})
 
 	AfterEach(func() {
-		testhelpers.K8sDeleteWithRetry(k8sClient, ctx, &instance)
+		objKey := client.ObjectKey{Namespace: Namespace, Name: instance.Name}
+		testhelpers.K8sDeleteWithRetry(k8sClient, ctx, objKey, &instance)
 		createdBackups := &v1alpha1.BackupList{}
 		Expect(k8sClient.List(ctx, createdBackups)).Should(Succeed())
 		for _, backup := range createdBackups.Items {
-			testhelpers.K8sDeleteWithRetry(k8sClient, ctx, &backup)
```

Comment:
in golang, loop variable is reused (see [here](https://evan-jones.appspot.com/go-gotcha-loop-variables.html)), so here backup can change and &backup might cause the flakes, change to `for i, backup ... and k8sDeleteWithRetry(... &createdBackups.Items[i])` might help

Reply:
Thanks for the comment, but I don't think this is necessary. 
In this case we use loop variable exactly as designed, just for use in one loop cycle, K8sDeleteWithRetry doesn't persist or store the loop variable anywhere else.

Reply:
LGTM